src/events/SDL_mouse.c

```diff
@@ -243,14 +243,6 @@ SDL_UpdateMouseFocus(SDL_Window * window, int x, int y, Uint32 buttonstate, SDL_
     SDL_Mouse *mouse = SDL_GetMouse();
     SDL_bool inWindow = SDL_TRUE;
 
-    if (window && ((window->flags & SDL_WINDOW_MOUSE_CAPTURE) == 0)) {
```

Comment:
I feel in this case if we just comment out it would be easier to future diff with SDL lib, and we know that we didn't accidentally remove it. And if we comment out, a comment with a link to the reason why we do that, such as the pull request and and the Asana ticket, wherever the actual explanation/context is, will serve future readers.